src/smalloc.cc

```diff
@@ -152,19 +166,9 @@ size_t ExternalArraySize(enum ExternalArrayType type) {
 void CopyOnto(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
-  if (!args[0]->IsObject())
-    return env->ThrowTypeError("source must be an object");
-  if (!args[2]->IsObject())
-    return env->ThrowTypeError("dest must be an object");
-
   Local<Object> source = args[0].As<Object>();
   Local<Object> dest = args[2].As<Object>();
 
-  if (!source->HasIndexedPropertiesInExternalArrayData())
-    return env->ThrowError("source has no external array data");
-  if (!dest->HasIndexedPropertiesInExternalArrayData())
```

Comment:
Ditto on the `Assert()`. And pretty much anywhere else that the checks have been removed.